src/js/helpers/local_storage.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+var local_storage_session_length = 0;
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    domain = '';
+    excl_subdomains = false;
+
+    // `item` is an object which contains the original value
+    // as well as the time when it's supposed to expire
+    var item = {
+      value: this.encodeData( value ),
```

Comment:
❔ Because we are using a JavaScript object and we are setting `value` into `item.value`, I'm thinking we don't really need `this.encodeData` (and the corresponding `this.decodeData` when reading the value). I think we can just do this:

```suggestion
      value: value,
```

(Note: We can't use object short notation because "'object short notation' is available in ES6".)

In cookies.js file, `encodeData` and `decodeData` is needed to prevent character collision in the larger `document.cookie` string.